### DISCUSSION TITLE
AnyTone displaySettings backlightDurationRX=0 still turns on display on RX?

### DISCUSSION THREAD

**Discussion by ikbenkous** [hmatuschek/qdmr#572] [Q&A]
Radio: AnyTone AT-D878UVII Plus
Firmware version: 3.04

With `backlightDurationRX` set to `0`, the display still turns on when the radio receives.

This seems like a bug?

I hope you don't mind all the bug reports, @hmatuschek?

**Comment by hmatuschek** [comment]
> I hope you don't mind all the bug reports, @hmatuschek?

By no means. I like to have a large number of small issues to fix, rather a single large issue.

**Comment by hmatuschek** [comment]
In fact, this is not a bug. Encoding a duration of 0 implies infinite. This is true for the RX and TX backlight duration. I may add a note to the GUI.

**Reply by hmatuschek** [reply]
It is already there ...

https://github.com/hmatuschek/qdmr/blob/f10a412b85ab1e601c787bbf718f40af57fa4821/lib/anytone_extension.hh#L922-L923

but this one is wrong:

https://github.com/hmatuschek/qdmr/blob/f10a412b85ab1e601c787bbf718f40af57fa4821/lib/anytone_extension.hh#L927-L928

 * TX=0 implies always off
 * RX=0 implies always on

**Reply by hmatuschek** [reply]
Should be fixed in PR #573.
